### ISSUE TITLE
reading 16bit tiff files 

### ISSUE THREAD

**Issue by ogre** [DigitalSlideArchive/tifftools#83]
I have a 16 bit tiff file (150mb, 6k resolution) that fails to load. Mem usage just goes up until I kill python process at around 20Gb.
Is reading 16 bit tiffs supported?

I am under windows 10

**Comment by manthey** [comment]
Yes, 16 bit tiffs are definitely supported.  Can you share the file?  If not, if you run `tifftools dump -vvvv <file>` it should print what it is doing and we'd get an idea of where it is having an issue.

**Comment by ogre** [comment]
Ok, here are two files, that should be "identical" in structure (two brackets from multibracket set).
They both come from dcraw.
ok.tiff loads OK
fail.tiff does not :)

link (~300mb) https://we.tl/t-XsKbY3F5Db

**Comment by manthey** [comment]
So these files aren't actually conformant to the tiff specification.  Both have an EXIF tag block which then claims to have another tag block at an address that ISN'T a tag block (it is within the image data).  It also looks like the same is done with the GPS data tag block (it claims to have a second such block also at a nonsense location).  While trying to parse the image data as tag blocks, the program consumes a lot of memory.  

If the latest version of dcraw is outputting these, then it has a bug that needs to be fixed.

**Comment by ogre** [comment]
Thank you !
